src/amuse/community/crater/src/test.f90

```diff
@@ -0,0 +1,4 @@
+FUNCTION echo(input)
+    INTEGER echo, input
+    echo = input
+END FUNCTION
```

Comment:
this is probably a remnant from the directory stub?

Reply:
certainly seems that way, I'll delete it